src/gallium/drivers/lima/lima_resource.c

```diff
@@ -155,6 +155,23 @@ lima_resource_create(struct pipe_screen *pscreen,
    return pres;
 }
 
+static struct pipe_resource *
+lima_resource_create_with_modifiers(struct pipe_screen *pscreen,
+                                   const struct pipe_resource *templat,
+                                   const uint64_t *modifiers,
+                                   int count)
+{
+   struct pipe_resource tmpl = *templat;
+
+   /*
+    * We currently assume that all buffers allocated through this interface
```

Comment:
Why we assume this? Can't we just return NULL for not support or add a modifier for "tiled"?

Reply:
Just mimicking etnaviv behavior. If we don't do it here kmscube doesn't work.

Reply:
OK, we may improve it latter.